frontend/public/components/dashboard/dashboard-card/dashboard-card-body.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { CardBody } from '@patternfly/react-core';
+
+import { LoadingInline } from '../../utils';
+
+export class DashboardCardBody extends React.PureComponent<DashboardCardBodyProps> {
+  render() {
+    const { isLoading, classname, children, ...props } = this.props;
+    return (
+      <CardBody className={classNames('co-dashboard-card-body', classname)} {...props}>
+        {isLoading ? <LoadingInline /> : children}
```

Comment:
result of the url/prometheus fetch can be error https://github.com/openshift/console/pull/1591/files#diff-7de87f9c74c975914fd0aeb9382a9db3R47 and every card handles that error on its own. ie. if the card cannot parse the result (prometheus pod crashed and fetch resulted in Error) it will show `Not Availalbe` as described here https://docs.google.com/document/d/1q0DIcUKgFhvNbtGt-SMl35Z2jHA_QO2r7aaBuFmwE2U/edit#heading=h.4s0i6ahqdw12